oid_validation/pulp/oid_validation/oid_validation.py

```diff
@@ -29,14 +30,33 @@ def authenticate(environ, config=None):
     '''
     Framework hook method.
     '''
-    cert_pem = environ["mod_ssl.var_lookup"]("SSL_CLIENT_CERT")
+    # An output stream (file-like object) to which error output can be written.
+    # For many servers, this will output to the main error log. For more
+    # information, see PEP333.
+    wsgi_error_logger = environ["wsgi.errors"].write
```

Comment:
I think this may get written to the httpd logs. Is that right? If so, is that the behavior we want? I guess this is a separate process from Pulp, so it doesn't have the Pulp logger set up. Maybe we could move the logging code to pulp.common if it would be useful to other bits than just pulp.server…? Or this is fine too…